sdk/batch/azure-batch/tests/batch_preparers.py

```diff
@@ -100,15 +104,11 @@ def create_resource(self, name, **kwargs):
             credentials = AzureNamedKeyCredential(keys.account_name, keys.primary)
```

Comment:
This is not part of your changes but I just wanted to call out that when I first created testcases for CSharp I choose to use AzureNamedKeyCredentials but this causes issues in playback where values or timestamps didn't match and caused failures.  Now I'm not sure if support layers are similar between CSharp and Python but CSharp has a system in place to ensure on playback auth works.  I switched over to using TokenCredential, see this CSharp doc https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core.TestFramework/README.md#tokencredential.  FYI, Here's my test code https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/batch/Azure.Compute.Batch/tests/Infrastructure/BatchLiveTestBase.cs#L63C55-L63C81